tests/llmcompressor/transformers/sparsification/test_compress_tensor_utils.py

```diff
@@ -171,34 +179,41 @@ def test_quant_model_reload(format, dtype, tmp_path):
         oneshot_device=device,
         precision=dtype,
     )
-
-    model = SparseAutoModelForCausalLM.from_pretrained(
-        tmp_path / "oneshot_out", torch_dtype=dtype
+    save_path = tmp_path / "oneshot_out"
+    model.save_pretrained(
+        save_path,
+        quantization_format=format,
+        save_compressed=True,
```

Comment:
this seems like we're now changing what the test was testing before?

Reply:
Previously oneshot pathway does the save, so this is to replicate that behavior